README.md

```diff
@@ -1,26 +1,60 @@
-# Offscreen Effect Player module
+# [Offscreen Effect Player module (OEP-module)](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
+
+[Description of the **Offscreen Effect Player** on the Banuba SDK website](https://docs.banuba.com/face-ar-sdk-v1/core/cpp_api_overview#offscreen-effect-player)
 
 ## Description
 
-Common submodule for some projects using **offscreen_effect_player**
-The module implements interaction with the effect player. The module consists of six main interfaces:
+Common submodule for projects integrating Banuba SDK using C++. This module is independent of the Banuba SDK and represents the implementation of interfaces for  [**Offscreen effect player**](./interfaces/offscreen_effect_player.hpp). The module is a wrapper over Banuba SDK interfaces and provides necessary interfaces for image processing including postprocessing (orientation change, format conversion), and management of effects. This module demonstrates one of the approaches and simplifies integration to the application.
+
+## Folders structure
+
+- [**docs**](./docs/) - illustrations with screenshots and images
+- [**interfaces**](./interfaces/) - offscreen effect player interfaces
+- [**offscreen_effect_player**](./offscreen_effect_player/) - contains the implementation of the **offscreen_effect_player**, **image_processing_result** and **pixel_buffer** intefaces. The implementation of **offscreen_effect_player** manages rendering via **ofscreen_render_target** interface and manages **effect_player**
+- [**offscreen_render_target**](./offscreen_render_target/) - is an implementation option for the **offscreen_render_target** interface. Allows to prepare gl framebuffers and textures for receiving a frame from gpu, receive bytes of the processed frame from the gpu and pass them to the cpu, as well as, if necessary, set the orientation for the received frame. This implementation uses GLFW to work with gl context
+- [**opengl**](./opengl/) - OpenGL utilities
+- [**third**](./third/) - third party libraries
+
+## Description of interfaces
+
+The module consists of six main interfaces.
```

Comment:
The module by itself implements a piece of image processing conveyor:  it accepts image for processing, confgures the offscreen rendering context, pass the image further to the main rendering engine - the Banuba SDK, and after provides the capabilities for image postprocessing. Based on this the module implements only some of the interfaces which do not introduce specifics of a particular API, e.g. Banuba SDK supports C++ API (link) and C API (link), the module doesn't know which API is used for the Banuba SDK management that is why **effect_player** interface is left unimplemented and its implementation is provided by the application.